org.eclipse.scanning.device.ui/src/org/eclipse/scanning/device/ui/points/ExecuteView.java

```diff
@@ -495,8 +497,12 @@ private void update(IProgressMonitor monitor) {
 		        	styledString.append(getScanRegions(cm.getRegions()), StyledString.QUALIFIER_STYLER);
 
 					if (monitor.isCanceled()) return;
-		        	styledString.append("\nMonitors: ");
-		        	styledString.append(getMonitorNames(), StyledString.DECORATIONS_STYLER);
+		        	styledString.append("\nPer point monitors: ");
+		        	styledString.append(getMonitorNames(MonitorRole.PER_POINT), StyledString.DECORATIONS_STYLER);
+
+					if (monitor.isCanceled()) return;
+		        	styledString.append("\nPer scan monitors: ");

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This line will not be executed conditionally; only the first line of this 2-line block will be. The rest will execute unconditionally. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2681)